sdk/search/azure-search-documents/CHANGELOG.md

```diff
@@ -4,12 +4,43 @@
 
 ### Features Added
 
+- Added knowledge agents operations in `SearchIndexClient`.
+- Added `resync` method in `SearchIndexerClient`.
+- Exposed `@search.reranker_boosted_score` in the search results.
+
+- Added new models:
+    - `azure.search.documents.QueryResultDocumentInnerHit`
+    - `azure.search.documents.indexes.ChatCompletionExtraParametersBehavior`
+    - `azure.search.documents.indexes.ChatCompletionResponseFormat`
+    - `azure.search.documents.indexes.ChatCompletionResponseFormatType`
+    - `azure.search.documents.indexes.ChatCompletionResponseFormatJsonSchemaProperties`
+    - `azure.search.documents.indexes.ChatCompletionSchema`
+    - `azure.search.documents.indexes.ChatCompletionSkill`
+    - `azure.search.documents.indexes.CommonModelParameters`
+    - `azure.search.documents.indexes.DocumentIntelligenceLayoutSkillChunkingProperties`
+    - `azure.search.documents.indexes.DocumentIntelligenceLayoutSkillChunkingUnit`
+    - `azure.search.documents.indexes.DocumentIntelligenceLayoutSkillExtractionOptions`
+    - `azure.search.documents.indexes.DocumentIntelligenceLayoutSkillOutputFormat`
+    - `azure.search.documents.indexes.IndexerPermissionOption`
+    - `azure.search.documents.indexes.IndexerResyncOption`
+    - `azure.search.documents.indexes.KnowledgeAgent`
+    - `azure.search.documents.indexes.KnowledgeAgentAzureOpenAIModel`
+    - `azure.search.documents.indexes.KnowledgeAgentModel`
+    - `azure.search.documents.indexes.KnowledgeAgentModelKind`
+    - `azure.search.documents.indexes.KnowledgeAgentRequestLimits`
+    - `azure.search.documents.indexes.KnowledgeAgentTargetIndex`
+    - `azure.search.documents.indexes.PermissionFilter`
+    - `azure.search.documents.indexes.RankingOrder`
+    - `azure.search.documents.indexes.SearchIndexPermissionFilterOption`
```

Comment:
I see that `KnowledgeAgentRetrievalClient` and various model classes in `azure.search.documents.agent.models` are added and public as well. Were these intended to be exposed? If so, should make an entry here for them.